frontend/class-opengraph-oembed.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Frontend
+ */
+
+/**
+ * Class WPSEO_OpenGraph_OEmbed
+ *
+ * This class handles the transformation of oEmbed JSON/XML to utilize WPSEO's OpenGraph
+ * settings.
+ *
```

Comment:
I think that you should remove this line.

Reply:
removed